src/test/unit/controls/Timeline/Timeline-spec.js

```diff
@@ -217,6 +217,30 @@ describe("Timeline", () => {
             expect(timeline.contentConfig.length).toBe(1);
             expect(timeline.contentConfig[0].config.key).toBe(data.key);
         });
+        it("Applies custom padding to the timeline instead of default padding", () => {
+            const data = getData(valuesJSON);
+            const input = getAxes(axisJSON);
+            input.padding = {
+                left: 10,
+                right: 20
+            };
+            timeline = new Timeline(input);
+            timeline.loadContent(data);
+            input.bindTo = "";
+            input.axis = {};
+            input.content = null;
+            data.tasks = [];
+            const cfg = timeline.config;
+            expect(input.bindTo).toBe("");
+            expect(input.axis).toEqual({});
+            expect(data.tasks).toEqual([]);
+            expect(cfg.bindTo).toBe("#testCarbonTimeline");
+            expect(cfg.axis.x.lowerLimit).toBe(axisJSON.x.lowerLimit);
+            expect(cfg.axis.x.upperLimit).toBe(axisJSON.x.upperLimit);
+            expect(cfg.padding.hasCustomPadding).toBe(true);
```

Comment:
The test doesnt check for the padding property. Neither does it check if the content width and height properties have changed.

Reply:
Updated in 763c968